src/Lykke.Service.LP3.DomainServices/Lp3Service.cs

```diff
@@ -1,26 +1,26 @@
 using System;
-using System.Collections.Concurrent;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
 using Autofac;
 using Common;
 using Common.Log;
+using JetBrains.Annotations;
 using Lykke.Common.Log;
 using Lykke.Service.LP3.Domain;
 using Lykke.Service.LP3.Domain.Assets;
 using Lykke.Service.LP3.Domain.Exchanges;
 using Lykke.Service.LP3.Domain.Orders;
 using Lykke.Service.LP3.Domain.Services;
 using Lykke.Service.LP3.Domain.Settings;
-using Lykke.Service.LP3.Domain.TradingAlgorithm;
 
 namespace Lykke.Service.LP3.DomainServices
 {
     public class Lp3Service : ILp3Service, IStartable, IDisposable
```

Comment:
Could you move SynchronizeAsync to the wrapper like in NE project?

Reply:
I think it's unnecessary in the current state of LP3 project to extract synchronisation logic into separate wrapper class as there is only one class with such sync logic and this is just an inner implementation of it.